build.sbt

```diff
@@ -133,76 +114,73 @@ lazy val metals = project
     testWorkspace % "test->test"
   )
 
+lazy val `sbt-metals` = project
+  .settings(
+    sbtPlugin := true,
+    crossScalaVersions := List(V.scala212, V.scala210),
+    sbtVersion in pluginCrossBuild := {
+      scalaBinaryVersion.value match {
+        case "2.10" => "0.13.17"
+        case "2.12" => "1.0.4"
+      }
+    },
```

Comment:
are you replicating the `^` logic by hand? Was it broken?

Reply:
I get it now, thanks :)

Reply:
The primary motivation was to use `+` instead of `^`, it's conceptually easier IMO to have to only think about the Scala version, especially when it comes to publishing all modules. I think sbt doesn't work this way by default in order to be able to hypothetically support multiple sbt versions on the same Scala version.